src/clib/cool1d_multi_g.F

```diff
@@ -653,6 +978,36 @@ subroutine cool1d_multi_g(
 !     --- Cooling from HD ---
 
       if (ispecies .gt. 2) then
+#define USE_CHIAKI_WISE2019
+#ifdef USE_CHIAKI_WISE2019
+      do i = is+1, ie+1
+      if ( itmask(i) ) then
+
+         lognhat = logHDI(i) - logdvdr(i)
+
+         call interpolate_3D_g(
+     &     lognhat, logT(i), logH(i), int8(LHD_N), 
+     &     LHD_D, LHD_dD, LHD_T, LHD_dT, LHD_H, LHD_dH,
+     &     int8(LHD_Size), LHD_L, log_Linv)
+         L = 1.d1**(-log_Linv)
+
+         if (icmbTfloor .eq. 1) then
+         call interpolate_3D_g(
+     &     lognhat, logTcmb(i), logH(i), int8(LHD_N), 
+     &     LHD_D, LHD_dD, LHD_T, LHD_dT, LHD_H, LHD_dH,
+     &     int8(LHD_Size), LHD_L, log_Ginv)
+         G = 1.d1**(-log_Ginv)
+         else
+         G = tiny8
+         endif
+
+         LHD(i) = - (L - G) / dom * HDI(i,j,k)/3.d0
+         if (LHD(i) .ne. LHD(i)) LHD(i) = 0.d0
+         edot(i) = edot(i) + LHD(i) !!!! CAUTION !!!!
```

Comment:
Can you maybe elaborate on this ``!!!! CAUTION !!!!`` comment that's here?

Reply:
I've checked this part. Just forgot to delete this. So we can delete `!!!! CAUTION !!!!`.